requirements.txt

```diff
@@ -2,7 +2,10 @@
 click==8.1.3
 coverage==7.2.7
 grablib==0.8
+watchfiles==0.19.0
+devtools==0.11.0
```

Comment:
These are already installed from setup.py. But, we often duplicate them here to help test specific against specific versions, so fine with me.